addons/calendar/models/res_partner.py

```diff
@@ -61,10 +62,13 @@ def get_attendee_detail(self, meeting_ids):
         """
         attendees_details = []
         meetings = self.env['calendar.event'].browse(meeting_ids)
-        meetings_attendees = meetings.mapped('attendee_ids')
+        all_attendees = meetings.mapped('attendee_ids')
+        attendees_by_partner = defaultdict(lambda: self.env['calendar.attendee'])
+        for attendee in all_attendees:
+            attendees_by_partner[attendee.partner_id] += attendee
```

Comment:
Honestly, I find it confusing to sometimes use the record as dict key and sometimes the id as dict key.
I believe it would be clearer to stick to using record ids as keys, and avoid records as much as possible ;).